arcade/window_commands.py

```diff
@@ -16,7 +16,7 @@
 
 if TYPE_CHECKING:
     from arcade import Window
-
+    from arcade.application import View
```

Comment:
This one should ideally be further up with the other imports. Then do a `python make.py format` to ensure isort is happy.

Reply:
When I tried this it caused a circular import error because application imports from window_commands

Reply:
Wait whoops that was the other file! I'll change this

Reply:
Oh actually I think it was this one